packages/wrangler/src/api/dev.ts

```diff
@@ -13,39 +14,73 @@ interface DevOptions {
 	siteInclude?: string[];
 	siteExclude?: string[];
 	nodeCompat?: boolean;
+	compatibilityDate?: string;
 	experimentalEnableLocalPersistence?: boolean;
-	_: (string | number)[]; //yargs wants this
-	$0: string; //yargs wants this
+	liveReload?: boolean;
+	watch?: boolean;
+	vars: {
+		[key: string]: unknown;
+	};
+	kv?: {
+		binding: string;
+		id: string;
+		preview_id?: string;
+	}[];
+	durableObjects?: {
+		name: string;
+		class_name: string;
+		script_name?: string | undefined;
+		environment?: string | undefined;
+	}[];
+	showInteractiveDevSession?: boolean;
+	logLevel?: "none" | "error" | "log" | "warn" | "debug";
+	logPrefix?: string;
+	inspect?: boolean;
+	forceLocal?: boolean;
+	cfFetch?: boolean;
+	miniflareCLIOptions?: MiniflareCLIOptions;
```

Comment:
I think this shouldn't be so generic named, let's just use enableAssetsServiceBinding as the prop to pass ( and it should probably be called `enablePagesAssetsServiceBinding`

Reply:
Yup, happy to change this if you prefer it. I had specifically tried to make it something more general that could be re-used, as I figured there's probably only going to be more things like this, but can easily make it a Pages-specific thing right now.